facebook/delphiFacebook/R/variables.R

```diff
@@ -292,7 +328,8 @@ code_testing <- function(input_data) {
     input_data$t_tested_reason_crowd <- is_selected(testing_reasons, "6")
     input_data$t_tested_reason_visit_fam <- is_selected(testing_reasons, "7")
     input_data$t_tested_reason_other <- is_selected(testing_reasons, "8")
-    
+    input_data$t_tested_reason_travel <- is_selected(testing_reasons, "9")
```

Comment:
Some of these testing reasons were removed in Wave 11, and as written, the code will just show their estimates go to 0 after Wave 11 is deployed. Should we change the code so they're marked NA for Wave 11+, rather than `is_selected` just reporting that they're not selected?